From 84b6e5c7aee44a052bddffca9e7d486eb97c50f7 Mon Sep 17 00:00:00 2001
From: FabioLolix <fabio.loli@disroot.org>
Date: Sun, 11 Jun 2023 19:39:37 +0200
Subject: [PATCH] Fix build with gcc 13.1

---
 types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/types.h b/types.h
index 07f78a9..84cf3cb 100644
--- a/types.h
+++ b/types.h
@@ -39,6 +39,7 @@
 #include <vector>
 #include <fstream>
 #include <array>
+#include <cstdint>
 
 class Error : public std::runtime_error
 {
